pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/NamespaceService.java

```diff
@@ -822,13 +825,42 @@ public CompletableFuture<Void> splitAndOwnBundle(NamespaceBundle bundle, boolean
         return unloadFuture;
     }
 
+    public Map<String, TopicStatsImpl> getTopicStats(NamespaceBundle bundle) {
+        ConcurrentOpenHashMap<String, Topic> topicMap = pulsar.getBrokerService().getMultiLayerTopicMap()
+        .computeIfAbsent(bundle.getNamespaceObject().toString(), k -> {
```

Comment:
we should modify the multiLayerTopicMap returned by the BrokerService.
this part that accesses that data structure should be inside BrokerService, otherwise we are breaking incapsulation

Reply:
Fixed.PTAL,thanks! @eolivelli